less/jumbotron.less

```diff
@@ -9,6 +9,27 @@
   background-color: @jumbotron-bg;
   transition: background-color 0.2s ease-in;
 
+  &.bg-focus {
+    h1, .h1 {
+      color: @brand-navy;
+    }
+
+    .lead {
+      color: @brand-grey-plus-10;
+    }
+  }
```

Comment:
mmh not sure, i've added .bg focus as a way to keep the default jumbotron to it's current styles but just enforce a white bg

happy to move the default jumbotron styles to this then (plus migrate old styles onto navy bg + white font)